src/utils/types.ts

```diff
@@ -9,13 +7,3 @@ export type Message = {
   payload?: any;
   state?: any;
 };
-
-export type Options = Parameters<typeof useStore>[0];
```

Comment:
Note that this was more precise. Now, you basically copied it, it doesn't follow when Jotai's `useStore` changes its option.